Src/WitsmlExplorer.Api/Query/BhaRunQueries.cs

```diff
@@ -1,8 +1,10 @@
 using System;
 using System.Globalization;
+
```

Comment:
System, system.globalization and witsml.explorer.api.models are unused. Also unnecessary spacing